packages/capacitor-notifications/src/definitions.ts

```
@@ -0,0 +1,179 @@
+  channel?: string
+}
+
+export interface CapgoUpdateCheckResult {
+  status: 'disabled' | 'unavailable' | 'no_update' | 'installed' | 'failed'
+  version?: string
+  bundleId?: string
+  error?: string
+}
+
+export interface CapgoNotificationsNativePlugin {
+  checkPermissions(): Promise<CapgoNotificationPermissionStatus>
```

ESLint

[error] Shorthand method signature is forbidden. Use a function property instead.

(ts/method-signature-style)